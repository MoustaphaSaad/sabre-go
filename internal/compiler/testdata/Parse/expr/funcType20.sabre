func(5 int)