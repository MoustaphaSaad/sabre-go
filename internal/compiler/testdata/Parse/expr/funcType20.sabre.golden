>> 	func(5 int)
>> 	     ^      
Error[internal/compiler/testdata/Parse/expr/funcType20.sabre:1:6]: expected an identifier
